### DISCUSSION TITLE
Regulatory element nomenclature

### DISCUSSION THREAD

**Discussion by ahwagner** [HGVSnomenclature/hgvs-nomenclature#16] [Other discussion topics]
How should regulatory elements be described? This is a question for the HUGO nomenclature committees broadly; it may not apply specifically to HGVS.

The current VICC nomenclature for regulatory elements is [here](https://fusions.cancervariants.org/en/latest/nomenclature.html#regulatory-nomenclature).

**Current Status:**
Awaiting formal vote by HVNC

**Comment by murphyte** [comment]
FYI, RefSeq now includes a set of functional elements as part of its annotation, described in:
https://pubmed.ncbi.nlm.nih.gov/34876495/

If there are features of clinical interest, do let us know and we can see about annotating them. The dataset is focused on experimentally-validated features and not those only inferred from epigenomic signatures. More high-throughout experimental studies are coming out, so the dataset is growing pretty rapidly.

One or more regulatory features are mapped within a biological_region feature that is associated with a GeneID (we consider them non-genic, but have leveraged GeneIDs for annotation). So a regulatory element could be described relative to the coordinates of a biological_region feature, if you don't want to use straight NC coordinates. The notation would need some refinement.

**Reply by ahwagner** [reply]
I think these features are of general scientific interest. While our effort does have a strong clinical focus, we are also designing this notation to cover fusion descriptions for translational research.

We also see regulatory elements as non-genic, but recognize that they are often described as associated with a gene; our nomenclature uses "@" to help make this distinction. I wonder if there is room for other biological feature names there. It would require some use cases. Let me know when there is progress on this front, we would very much like more input on the regulatory element nomenclature design.

**Comment by elspethb** [comment]
Naming regulatory elements is something HGNC have been discussing for some years but there is currently no real push from the community to see them named in any systematic way. We will continue to monitor this and consider naming if things change in due course.

**Comment by murphyte** [comment]
For the RefSeq Functional Element (RefSeqFE) project, we're also not actively naming elements, other than incorporating published names where available (e.g. [HBA-LCR](https://www.ncbi.nlm.nih.gov/gene/106144573)). Most of the RefSeqFE regions are named with a generic LOC# symbol based on the assigned GeneID (most LOC#s are 9 digits). I think there's be marginal value in using a pattern like RSENH# (RefSeq Enhancer), where # would currently go up to >70000, and add complexity for elements with multiple activities. We're of course interested in making improvements to aid usability as the project evolves.
